README.md

```diff
@@ -1,3 +1,7 @@
+[![Build Status](https://dev.azure.com/shotgun-ecosystem/Toolkit/_apis/build/status/Engines/tk-maya?branchName=black)](https://dev.azure.com/shotgun-ecosystem/Toolkit/_build/latest?definitionId=28&branchName=black)
```

Comment:
Your badge says "black" branch. Curious how you got a bad for that instead of master.

Reply:
ah interesting, maybe I can check with you when your online.